conda.recipe/meta.yaml

```
@@ -24,7 +24,7 @@ requirements:
   run:
     - line_profiler
     - python
-    - spyder >=3
+    - spyder >=5
```

We should put 5.2 here
```suggestion
    - spyder >=5.2
```